Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookup.java

```diff
@@ -118,11 +118,14 @@ private void addAlternativeIds(String id, ConceptEntry entry) throws LuceneExcep
         if (entry.getAlternativeIds() == null) {
             entry.setAlternativeIds(new HashSet<String>());
         }
-        if (conceptTypesService.getConceptTypeByConceptId(id) == ConceptTypes.LOCAL_CONCEPT
-                || conceptTypesService.getConceptTypeByConceptId(id) == ConceptTypes.GENERIC_WORDNET_CONCEPT) {
+        if (conceptTypesService.getConceptTypeByConceptId(id) == ConceptTypes.LOCAL_CONCEPT) {
```

Comment:
why don't you say here: if concept is not conceptpower conecpt, add ids to alternative ids and then remove line 126 (since it's code duplication).